packages/falter-berlin-ssid-changer/files/lib/lib_ssid-changer.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/sh
+. /lib/functions.sh
+
+NODENAME=$(uci_get system @system[-1] hostname | cut -b -24 ) # cut nodename to not exceed 32 bytes
+OFFLINE_SSID="offline_""$NODENAME"
+
+check_connectivity() {
+    # check if we really lost internet connection.
+    local ERROR=0
+    ping -c1 8.8.8.8 > /dev/null || ERROR=$(($ERROR + 1))
+    ping -c1 1.1.1.1 > /dev/null || ERROR=$(($ERROR + 1))
+    ping -c1 9.9.9.9 > /dev/null || ERROR=$(($ERROR + 1))
+    echo "$ERROR"
+}
+
+get_interfaces() {
+    # get the names of every interface named *dhcp* and fetch its normal ssid. Thus we get
+    # 2.4 and/or 5 GHz both
+    local IFACES=$(uci show wireless | grep -e "dhcp.*\.ssid" | cut -d'=' -f1 )
```

Comment:
We should only get interfaces that match the freifunk pattern (`grep -e "dhcp.*\.ssid='.*\.freifunk.net'"`) here to avoid problems with additional private networks, especially since we have plans to add this functionality in the new wizard.

Reply:
That went into the most recent version too.